src/hb-ot-math.cc

```diff
@@ -85,8 +91,11 @@ hb_ot_math_get_constant (hb_font_t *font,
 
 /**
  * hb_ot_math_get_glyph_italics_correction:
- * @font: #hb_font_t from which to retrieve the value
- * @glyph: glyph index from which to retrieve the value
+ * @font: #hb_font_t to work upon
+ * @glyph: The glyph index from which to retrieve the value
+ *
+ * Fetches an italics-correction value (if one exists) for the specified
+ * glyph index.
  *
  * Return value: the italics correction of the glyph or 0
```

Comment:
```suggestion
  * Return value: the italics correction of the glyph or zero
```
For consistency with the change above.